docs/detections/prebuilt-rules/downloadable-packages/0-16-2/prebuilt-rule-0-16-2-searching-for-saved-credentials-via-vaultcmd.asciidoc

```diff
@@ -0,0 +1,85 @@
+[[prebuilt-rule-0-16-2-searching-for-saved-credentials-via-vaultcmd]]
+=== Searching for Saved Credentials via VaultCmd
+
+Windows Credential Manager allows you to create, view, or delete saved credentials for signing into websites, connected applications, and networks. An adversary may abuse this to list or dump credentials stored in the Credential Manager for saved usernames and passwords. This may also be performed in preparation of lateral movement.
```

Comment:
```suggestion
Windows Credential Manager allows you to create, view, or delete saved credentials for signing into websites, connected applications, and networks. An adversary may abuse this to list or dump credentials stored in the Credential Manager and gain access to usernames and passwords. This may also be performed in preparation for lateral movement.
```